main/acle.md

```diff
@@ -9001,9 +9001,21 @@ following it. --><span id="__arm_za_disable"></span>
 
 The intrinsics in this section have the following properties in common:
 
-*   Every argument named `tile`, `slice_offset` or `tile_mask` must
-    be an integer constant expression in the range of the underlying
-    instruction.
+*   Every argument named `tile` or `tile_mask` must be an integer constant
+    expression in the range of the underlying instruction.
+
+*   Some SME instructions identify a slice of ZA using the sum of a 32-bit
+    general-purpose register and an immediate offset.  The intrinsics for
+    these instructions have a 32-bit argument called `slice`, which is
```

Comment:
To be consistent, change "called" to "named" - as you did in the opening paragraph